package.json

```diff
@@ -38,7 +38,8 @@
     "lerna": "^3.20.2",
     "lockfile-lint": "^4.1.0",
     "mock-fs": "^4.7.0",
-    "prop-types": "*"
+    "prop-types": "*",
+    "regenerator-runtime": "^0.13.7"
```

Comment:
why are the `babel.config.js` and "regenerator-runtime" files needed at the top level of the monorepo?

Reply:
This would enable us to support ES6 import/export, and we can use it anywhere across the other packages too